### ISSUE TITLE
Provide health check for Docker image

### ISSUE THREAD

**Issue by novoj** [FgForrest/evitaDB#550]
To reliably run evitaDB on production systems, we need to provide a health check for the Docker container. The container should be healthy if no OutOfMemoryError occurred and the endpoint responds within 2s with 2 retries.

**Comment by lukashornych** [comment]
Doesn't this solve the `alive` endpoints we provide in GQL and REST endpoint? Should this do something more?

**Comment by novoj** [comment]
Yes, I'm implementing it on the "system" endpoint so we don't have to rely on APIs that could be disabled at any time. I will also detect the situations when queues are full and requests are rejected, also when GC cannot free memory and the system will crash due to insufficient memory, and also when there are a lot of internal errors that might signal some problem. This is not covered by our "liveness" probes in the GQL / REST APIs.

**Comment by novoj** [comment]
We should also prepare readiness probe. See [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)

**Comment by novoj** [comment]
Documented here: https://evitadb.io/documentation/operate/monitor?lang=evitaql#readiness-and-liveness-probes
